plugins/quetz_sql_authenticator/quetz_sql_authenticator/cli.py

```diff
@@ -0,0 +1,83 @@
+import click
```

Comment:
It would be nice if you could do these CRUD operations through REST, instead of using a CLI tool.

Reply:
Makes sense 👍🏼 Vaguely related: do we want to couple Quetz users and the credentials created by our plugin more closely? E.g. by cascading on delete and creating the API key when the credentials are created as opposed to on the first login?

Reply:
Currently the user info or credentials persist when deleting the other. This makes deleting a `user` very unintuitive imo, because logging in again will work and will generate a new API key. Maybe there's a good reason to do it like this though.

Reply:
Makes sense. How do you currently create the API keys for technical users? Through the REST API? It seems to me that it's not automatically created on login.

Reply:
> How do you currently create the API keys for technical users?

Get is a "Get API Key" button on the Quetz server's home page. I believe this does a `POST /api/api-keys` request under the hood. 

> It seems to me that it's not automatically created on login.

To the best of my knowledge, API keys are not created automatically upon login.